content/articles/how-sweet-is-kubernetes/index.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: engineering-education
+status: publish
+published:true 
+url: /engineering-education/how-sweet-is-kubernetes/
+title: How sweet is Kubernetes?
+description: .
```

Comment:
Write here a brief description of what your article is about.

Reply:
> Write here a brief description of what your article is about.

How sweet is Kubernetes is an article which is targeted at simplifying concepts of Kubernetes for engineering students and aspiring DevOps professionals. It started with demystifying the basic concept of micro-service using Fleetman Tracking Application which is built on a simple micro-service architecture, using it as a tool to further explain the concepts of Kubernetes in the simplest manner. The tutorial will serve as a good foundation for beginners and also a means of enlightenment for intermediate-level developers and junior DevOps engineers.
Grab a cup of coffee and enjoy the wonderful piece of work.